-------------------------------------------------------------------------------
Test set: com.Nttadta.nttdatacentersspringt1SLA.NttdatacentersSpringT1SlaApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 12.386 s <<< FAILURE! - in com.Nttadta.nttdatacentersspringt1SLA.NttdatacentersSpringT1SlaApplicationTests
contextLoads  Time elapsed: 0.018 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext for [WebMergedContextConfiguration@66297b63 testClass = com.Nttadta.nttdatacentersspringt1SLA.NttdatacentersSpringT1SlaApplicationTests, locations = [], classes = [com.Nttadta.nttdatacentersspringt1SLA.NttdatacentersSpringT1SlaApplication], contextInitializerClasses = [], activeProfiles = [], propertySourceLocations = [], propertySourceProperties = ["org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true"], contextCustomizers = [org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@3569fc08, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@7d42c224, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@315f43d5, org.springframework.boot.test.autoconfigure.actuate.observability.ObservabilityContextCustomizerFactory$DisableObservabilityContextCustomizer@9da1, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@4a11eb84, org.springframework.boot.test.context.SpringBootTestAnnotation@decebd39], resourceBasePath = "src/main/webapp", contextLoader = org.springframework.boot.test.context.SpringBootContextLoader, parent = null]
Caused by: java.lang.IllegalStateException: Failed to execute CommandLineRunner
Caused by: org.springframework.dao.IncorrectResultSizeDataAccessException: query did not return a unique result: 2
Caused by: jakarta.persistence.NonUniqueResultException: query did not return a unique result: 2

